packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/selectedJob.st

```diff
@@ -0,0 +1,4 @@
+accessing
+selectedJob
+	
+	^ selectedJob ifNil: [ selectedJob := 0 ]
```

Comment:
```suggestion
	^ selectedJob ifNil: [ selectedJob := -1 ]
```
I am not sure whether this is 0- or 1-indiced, either way if this should be an invalid result, make it -1 to be sure it is interpreted as such.

If not beware of the syntactical meaning of this: In case no jobs are selected (and for example an action is run), it will be performed for the first job (unless the selection of jobs is the trigger for any action that can be run)